generator/Makefile

```diff
@@ -34,18 +37,31 @@ all: mibs
 clean:
 	rm -v \
 		$(MIBDIR)/* \
-		$(MIBDIR)/.cisco_v2 \
+		$(MIBDIR)/cisco_v2 \
 		$(MIBDIR)/.paloalto_panos \
 		$(MIBDIR)/.synology
 
+generator:
+	go build
+
+generate: generator mibs
+	MIBDIRS=$(MIB_PATH) ./generator generate
+
+parse_errors: generator mibs
+	MIBDIRS=$(MIB_PATH) ./generator parse_errors
```

Comment:
You know this will exit 0 even if there are parse errors?

Reply:
Yup, it's just a shortcut for manual testing and visibility at this point. I was going to open an issue to add a "fail on errors" flag to the generator so we can include it as an actual part of the tests.

Reply:
I added the flag to allow failure code on parse errors.